http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1225,16 +1225,30 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
 
       // Build delete namespace request
       jobId = UUID.randomUUID()
-      deleteNamespaceRequest = new DeleteControlledAzureResourceRequest().jobControl(
+      deleteResourceRequest = new DeleteControlledAzureResourceRequest().jobControl(
         new JobControl().id(jobId.toString)
       )
 
-      _ <- logger.info(ctx.loggingCtx)(s"WSM delete namespace request: ${deleteNamespaceRequest}")
+      _ <- logger.info(ctx.loggingCtx)(s"WSM delete resource request: ${deleteResourceRequest}")
 
       // Execute WSM call
-      result <- F.blocking(
-        wsmApi.deleteAzureKubernetesNamespace(deleteNamespaceRequest, workspaceId.value, wsmResource.resourceId.value)
-      )
+      result <- wsmResource.resourceType match {
```

Comment:
are there other types of `resourceType`'s we should be accommodating here?
this will be SO nice in an event-driven world - 🤓

Reply:
... dropping reference to [WsmResourceType](https://github.com/databiosphere/leonardo/blob/5409fb53e95f5ef69d88de4da69fca0d6704f597/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeControlledResourceComponent.scala#L67-L95) for convenience

Reply:
good question, these are the only 2 WSM resources for Apps that we poll on currently, we don't poll on `managedIdentity` so that deletion occurs in the call function, `deleteWsmResource`